sbnanaobj/StandardRecord/SRTrigger.h

```diff
@@ -10,23 +10,26 @@ namespace caf
   {
   public:
     SRTrigger();
-    u_int64_t global_trigger_time; ///< Time of trigger in UTC [ns]
-    u_int64_t beam_gate_time_abs; ///< Time of beam gate opening in UTC [ns]
-    double trigger_within_gate; ///< Time from  beam gate to the trigger [us]
-    double beam_gate_det_time; ///< Beam gate time on detector time scale [us]
+    u_int64_t global_trigger_time;  ///< Time of trigger in UTC [ns]
+    u_int64_t beam_gate_time_abs;   ///< Time of beam gate opening in UTC [ns]
+    double trigger_within_gate;     ///< Time from  beam gate to the trigger [us]
+    double beam_gate_det_time;      ///< Beam gate time on detector time scale [us]
     double global_trigger_det_time; ///< Trigger time on detector time scale [us]
 
     std::uint64_t prev_global_trigger_time; ///< Time of the last trigger in this stream in UTC [ns]
-    unsigned int source_type; ///< gate source type
-    unsigned int trigger_type; ///< trigger type
+    unsigned int source_type;               ///< Gate source type
+    unsigned int trigger_type;              ///< Trigger type
 
-    unsigned trigger_id; ///< global ID of the trigger 
-    unsigned gate_id; ///< global ID of the gate
+    unsigned trigger_id;  ///< Global ID of the trigger 
+    unsigned gate_id;     ///< Global ID of the gate
 
-    unsigned trigger_count; ///< count of trigger in this stream
-    unsigned gate_count; ///< count of gate in this stream
-    int gate_delta; ///< Number of gates since previous trigger in this stream (gate of this event included)
+    unsigned trigger_count; ///< Count of trigger in this stream
+    unsigned gate_count;    ///< Count of gate in this stream
+    int gate_delta;         ///< Number of gates since previous trigger in this stream (gate of this event included)
     
+    unsigned int trigger_logic_bits;   ///< Whether the trigger was from adder logic, PMT-majority logic or both, straight from the trigger hardware. Provided for the first cryostat in time.
+    unsigned int beam_to_trigger_time; ///< Nanoseconds from the beam gate opening to the first trigger in the cryostat, straight from the trigger hardware. Provided for the first cryostat in time.
```

Comment:
The reference time in here is the opening of the beam _gate_, not the beam arrival. I recommend to change the name to something more appropriate, e.g. `gate_to_trigger_time`.